Only in gnumeric-1.0.9: acinclude.m4
Common subdirectories: gnumeric-1.0.9.orig/autom4te.cache and gnumeric-1.0.9/autom4te.cache
Common subdirectories: gnumeric-1.0.9.orig/doc and gnumeric-1.0.9/doc
Only in gnumeric-1.0.9.orig/: gnumeric-config.h.in~
diff -u gnumeric-1.0.9.orig/gnumeric.desktop.in gnumeric-1.0.9/gnumeric.desktop.in
--- gnumeric-1.0.9.orig/gnumeric.desktop.in	Sat Jun  1 21:59:10 2002
+++ gnumeric-1.0.9/gnumeric.desktop.in	Sat Dec 20 18:28:58 2003
@@ -3,6 +3,8 @@
 _Comment=The GNOME Spreadsheet
 Exec=gnumeric %F
 Icon=gnome-gnumeric.png
-Terminal=0
+Terminal=false
 Type=Application
 Categories=Application;Office;Spreadsheet;
+Encoding=UTF-8
+# vi: encoding=utf-8
Common subdirectories: gnumeric-1.0.9.orig/icons and gnumeric-1.0.9/icons
Common subdirectories: gnumeric-1.0.9.orig/idl and gnumeric-1.0.9/idl
Common subdirectories: gnumeric-1.0.9.orig/intl and gnumeric-1.0.9/intl
Common subdirectories: gnumeric-1.0.9.orig/macros and gnumeric-1.0.9/macros
Common subdirectories: gnumeric-1.0.9.orig/omf-install and gnumeric-1.0.9/omf-install
Common subdirectories: gnumeric-1.0.9.orig/plugins and gnumeric-1.0.9/plugins
Common subdirectories: gnumeric-1.0.9.orig/po and gnumeric-1.0.9/po
Common subdirectories: gnumeric-1.0.9.orig/samples and gnumeric-1.0.9/samples
Common subdirectories: gnumeric-1.0.9.orig/src and gnumeric-1.0.9/src
Common subdirectories: gnumeric-1.0.9.orig/templates and gnumeric-1.0.9/templates
Common subdirectories: gnumeric-1.0.9.orig/tools and gnumeric-1.0.9/tools
